frontend/src/Settings/Quality/Definition/QualityDefinition.css

```diff
@@ -56,7 +56,7 @@
 .megabytesPerMinute {
```

Comment:
Slider Shading is off due to missing this add from the Radarr commit

![image](https://user-images.githubusercontent.com/376117/206826702-777fc80b-fbab-43fa-a5e9-e8321f679d1a.png)


![image](https://user-images.githubusercontent.com/376117/206826688-a23d393a-ff04-436e-92bd-3e42844ad457.png)

Reply:
Doh, saw this and didn't realize what it was doing, good catch.